index.html

```diff
@@ -21,16 +21,12 @@ <h1 class="post-title">
 
 <div class="pagination">
   {% if paginator.next_page %}
-    <a class="pagination-item older" href="{{ site.baseurl }}page{{paginator.next_page}}">Older</a>
+    <a class="pagination-item older" href="{{paginator.next_page_path | prepend: site.baseurl | replace: '//', '/' }}">Older</a>
```

Comment:
What is `replace: '//', '/'` for? I am assuming it is to remove extra slashes between `paginator.next_page_path` and `site.baseurl`. If so, I would recommend removing any trailing slashes from `site.baseurl` instead of replacing slashes (which could have unintended side effects). See [Jekyll's documentation for GitHub Pages](http://jekyllrb.com/docs/github-pages/).